app/src/main/res/layout/search.xml

```diff
@@ -9,18 +9,28 @@
               android:textSize="17sp"
               android:visibility="gone"
               android:textColor="?android:textColorPrimary" />
-    <Button android:id="@+id/btnGetTranslations" android:text="@string/menu_get_translations" 
+    <Button android:id="@+id/btnGetTranslations" android:text="@string/menu_get_translations"
     	android:layout_width="match_parent" android:layout_height="wrap_content" android:visibility="gone"/>
-    <TextView android:id="@+id/search_area"
-            android:textColor="?android:textColorPrimary"
-            android:textSize="17sp"
-            android:text="@string/search_key"
-            android:background="@android:drawable/title_bar"
-            android:layout_width="fill_parent"
-            android:layout_height="wrap_content" />
+
+  <androidx.appcompat.widget.SearchView
```

Comment:
#845 had an `EditText` here, I changed it to a `SearchView`.

Reply:
Not directly related, but IMHO this should actually go to toolbar menu.

Reply:
I agree (if only I'd thought of this before), I'm taking a look into this, I've got something mostly working, not the prettiest code at the moment though. I want to test more around it and see if I can improve the code first.

Reply:
@ozbek I've pushed f72bc5b and f82d4618.